android/BUILD.gn

```diff
@@ -6,6 +6,12 @@ declare_args() {
   brave_android_developer_options_code = ""
 }
 
+java_cpp_enum("brave_android_java_enums_srcjar") {
```

Comment:
@yachtcaptain23 Do you remember why this was added back as this was not part of the original ad notifications?